keyboards/subrezon/la_nc/keymaps/default/keymap.c

```diff
@@ -0,0 +1,94 @@
+    ),
+
+    [_SYM] = LAYOUT(
+    //.--------+--------+--------+--------+--------.                 .--------+--------+--------+--------+--------.
+        KC_LCBR, KC_AMPR, KC_ASTR, KC_LPRN, KC_RCBR,                   KC_MPRV, KC_PGUP,   KC_UP, KC_PGDN, KC_PIPE,
+    //|--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------|
+        KC_UNDS,  KC_DLR, KC_PERC, KC_CIRC, KC_RPRN,                   KC_MPLY, KC_LEFT, KC_DOWN, KC_RGHT, KC_LSFT,
+    //|--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------|
+        KC_TILD, KC_EXLM,   KC_AT, KC_HASH, KC_PLUS,                   KC_MNXT, KC_HOME,  KC_INS,  KC_END, KC_DQUO,
+    //|--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+         KC_TAB, KC_LALT, KC_LCTL,MO(_OPT),  KC_SPC,  KC_ESC, KC_LGUI, KC_BSPC, KC_LSFT, _______,  KC_DEL,  KC_ENT
+    //'--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
+    ),
+
+    [_OPT] = LAYOUT(
+    //.--------+--------+--------+--------+--------.                 .--------+--------+--------+--------+--------.
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------|
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------|
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+        XXXXXXX, XXXXXXX, XXXXXXX, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, XXXXXXX, XXXXXXX
+    //'--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
+    ),
+
+};
+
+#ifdef OLED_ENABLE
+
+bool oled_task_user(void) {
+       oled_write_P(PSTR("Layer:\n"), false);
+
+       switch (get_highest_layer(layer_state)) {
+              case _QWE :
+                     oled_write_P(PSTR("QWERTY\n"), false);
+                     break;
+
+              case _NUM :
+                     oled_write_P(PSTR("Numbers\n"), false);
+                     break;
+
+              case _SYM :
+                     oled_write_P(PSTR("Symbols\n"), false);
+                     break;
+
+              case _OPT :
+                     oled_write_P(PSTR("Options\n"), false);
+                     break;
+
+              default :
+                     oled_write_P(PSTR("Undefined\n"), false);
+       }
+
+       return false;
+};
+
+#endif
```

Comment:
Ideally, this block should go into a `la_nc.c` file, so that the keyboard can have a default behavior.  Eg, when compiling from the QMK Configurator.

Reply:
Done.